browser/src/control/jsdialog/Util.FocusCycle.js

```diff
@@ -63,5 +63,63 @@ function makeFocusCycle(container, failedToFindFocusFunc) {
 	});
 }
 
+function findFocusableElement(element, direction) {
+
+	// check if no element present the just return null
+	if (!element) return null;
+
+	// Check the current element if it is focusable
+	if (isFocusable(element)) return element;
+
+	// Check if sibling is focusable or contains focusable elements
+	const focusableInSibling = findFocusableWithin(
+		element,
+		direction,
+	);
+	if (focusableInSibling) return focusableInSibling;
+
+	// Depending on the direction, find the next or previous sibling
+	const sibling =
+		direction === 'next'
+			? element.nextElementSibling
+			: element.previousElementSibling;
+
+	if (sibling) {
```

Comment:
no need for this check, we can just return findFocusableElement as it returns null for non existing element